Initial pull request for aisera-datadog integration (#2039)
DataDog/integrations-extras: aisera/README.md

# Agent Check: Aisera

## Overview

 [Aisera's][1] domain-agnostic [AiOps][2] platform identifies complex behaviors and patterns while analyzing correlations and causality on alerts and incidents across applications, services, and systems. 
These advanced capabilities enable accurate prediction of outages and major incidents (MIs) while automating the task to find the root cause and executing runbooks for auto-remediation.

> The [Aisera's][1] integration enables you to ingest your Datadog alerts/metrics into Aisera.  Along with other historical and real-time data across your technology stack, Aisera [AiOps][2] automatically predicts major incidents and delivers proactive notifications in a conversational manner.

Reviewer: ```suggestion
The [Aisera][1] integration enables you to ingest your Datadog alerts/metrics into Aisera.  Along with other historical and real-time data across your technology stack, Aisera [AiOps][2] automatically predicts major incidents and delivers proactive notifications in a conversational manner.
```

      7. Impacted Storage (Any DB/Storage impacted)
      8. Impacted Users
      9. Incidents (Any Incident from ticketing systems which caused this major incident)
      10. Alerts (Datadog alerts/logs part of this major incident)
      11. Change Request (Any change request which caused this major incident)
      12. A timeline of the incident/alerts corelation.
      13. Root cause : A root cause generated by Aisera LLM model.
      14. Causal Graphs : Graph of business services
      15. Logs and Trance Analysis.
      16. Major Incident Connectivity Graph: A visual representation of how users are impacted by specific alerts/services/infrastructure.
      17. Score Over time : AMI scores overtime.
      18. Outage Probability over time.
      19. Major Incident Correlation Evidence
          ![my screenshot](images/aisera-major_incident_detail_1.png)
          ![my screenshot](images/aisera-major_incident_detail_2.png)
          ![my screenshot](images/aisera-major_incident_detail_3.png)
          ![my screenshot](images/aisera-major_incident_detail_4.png)
8. Once Aisera detects a major incident ,
   1. it will notify to the designated persona on the channel on their choice.

>    2. Aisera will create an incident on Datadog for Datadog Admins to have a visibility of the 

Reviewer 2: This sentence appears to be missing information at the end.

> 7. AiOps pipelines will generate major incidents/clusters based on the category and severioty of incidents recieved.
>    1. Each major incident contains the following main components.
>       1. Title : A brief info about the major incident 
>       2. Status : Open/Close.  
>       3. Cluster : Aisera generated cluster to which this major incident belongs to. 
>       4. Outage Probability: Outage Probablity for this major incident.
>       5. Impacted Business Services.
>       6. Impacted Systems (Any Virtual Machine/Pod impacted)
>       7. Impacted Storage (Any DB/Storage impacted)
>       8. Impacted Users
>       9. Incidents (Any Incident from ticketing systems which caused this major incident)
>       10. Alerts (Datadog alerts/logs part of this major incident)
>       11. Change Request (Any change request which caused this major incident)
>       12. A timeline of the incident/alerts corelation.
>       13. Root cause : A root cause generated by Aisera LLM model.
>       14. Causal Graphs : Graph of business services
>       15. Logs and Trance Analysis.
>       16. Major Incident Connectivity Graph: A visual representation of how users are impacted by specific alerts/services/infrastructure.
>       17. Score Over time : AMI scores overtime.
>       18. Outage Probability over time.
>       19. Major Incident Correlation Evidence
>           ![my screenshot](images/aisera-major_incident_detail_1.png)
>           ![my screenshot](images/aisera-major_incident_detail_2.png)
>           ![my screenshot](images/aisera-major_incident_detail_3.png)
>           ![my screenshot](images/aisera-major_incident_detail_4.png)
> 8. Once Aisera detects a major incident ,
>    1. it will notify to the designated persona on the channel on their choice.
>    2. Aisera will create an incident on Datadog for Datadog Admins to have a visibility of the 
> 9. We can also put Approvals in place so that before we approvals before any remediation step can run.
> 10. Aisera can also run any remediation task (Any API call/ Shell/Python/Powershell script/ Terraform manifests etc..) leveraging the No-code platform it provide.

Reviewer 2: ```suggestion
AIOps pipelines will generate major incidents/clusters based on the category and severity of incidents received.

Each major incident contains the following main components:
- **Title**: Brief info about the major incident
- **Status**: Open/Close
- **Cluster**: Aisera generated cluster that this major incident belongs to
- **Outage Probability**: Outage probability for this major incident
- **Impacted Business Services**
- **Impacted Systems**: Any Virtual Machine/Pod that was impacted
- **Impacted Storage**: Any DB/Storage that was impacted
- **Impacted Users**
- **Incidents**: Any Incident from ticketing systems that caused this major incident
- **Alerts**: Datadog alerts/logs that are part of this major incident
- **Change Request**: Any change request which caused this major incident
- A timeline of the incident/alerts correlation
- **Root Cause**: A root cause generated by the Aisera LLM model
- **Causal Graphs**: Graph of business services
- **Logs and Trace Analysis**
- **Major Incident Connectivity Graph**: A visual representation of how users are impacted by specific alerts/services/infrastructure
- **Score over Time**: AMI scores over time
- Outage Probability over Time
- **Major Incident Correlation Evidence**
          ![my screenshot](images/aisera-major_incident_detail_1.png)
          ![my screenshot](images/aisera-major_incident_detail_2.png)
          ![my screenshot](images/aisera-major_incident_detail_3.png)
          ![my screenshot](images/aisera-major_incident_detail_4.png)
          
Once Aisera detects a major incident:
- It notifies the designated persona on the channel of their choice.
- Aisera creates an incident in Datadog for Datadog Admins to have a visibility of the 

Aisera can require approvals before a remediation task can run.

Aisera can also run any remediation task (Any API call/ Shell/Python/Powershell script/ Terraform manifests etc..), leveraging the no-code platform it provide.
```

> ## Data Collection

Reviewer: ```suggestion
## Data Collected
```

> 1. One you run the datasource, Aisera will collect the metrices from datadog.

Reviewer: This will not render under the data collected tab
```suggestion
```

### Metrics

> Aisera can also collect the metrics and run AiOps pipeline on them.

Reviewer: ```suggestion
See [metadata.csv][6] for a list of metrics provided by this integration.

```

### Service Checks

Aisera does not include any service checks.

### Events

Aisera does not include any events.

## Troubleshooting

Need help? Contact [support@aisera.com](mailto:support@aisera.com).

[1]: https://aisera.com
[2]: https://aisera.com/products/aiops/
[3]: https://docs.datadoghq.com/account_management/api-app-keys/#application-keys
[4]: https://poc0.login.aisera.cloud/#YWlzZXJhLmV4dGVybmFsU3lzdGVtcy5leHRlcm5hbFN5c3RlbXNWaWV3

> [5]: https://aisera.com/contact/

Reviewer: ```suggestion
[5]: https://aisera.com/contact/
[6]: https://github.com/DataDog/integrations-extras/blob/master/aisera/metadata.csv
```

# Agent Check: Aisera

## Overview

[Aisera's][1] domain-agnostic [AIOps][2] platform identifies complex behaviors and patterns while analyzing correlations and causality on alerts and incidents across applications, services, and systems. 
These advanced capabilities enable accurate prediction of outages and major incidents (MIs) while automating the task to find the root cause and executing runbooks for auto-remediation.

> The [Aisera][1] integration enables you to ingest your Datadog alerts/metrics into Aisera.Along with other historical and real-time data across your technology stack, Aisera [AiOps][2] automatically predicts major incidents and delivers proactive notifications in a conversational manner.

Reviewer: ```suggestion
The [Aisera][1] integration enables you to ingest your Datadog alerts/metrics into Aisera. Along with other historical and real-time data across your technology stack, Aisera [AiOps][2] automatically predicts major incidents and delivers proactive notifications in a conversational manner.
```

## Setup

> 

Reviewer: Once your team implements OAuth for this integration, this setup section will need to be updated as well.

      - **Root Cause**: A root cause generated by the Aisera LLM model.
      - **Causal Graphs**: Graph of business services.
      - **Logs and Trace Analysis**
      - **Major Incident Connectivity Graph**: A visual representation of how users are impacted by specific alerts/services/infrastructure.
      - **Score over Time**: AMI scores over time.
      - **Outage Probability over Time**
      - **Major Incident Correlation Evidence**
          ![my screenshot](images/aisera-major_incident_detail_1.png)
          ![my screenshot](images/aisera-major_incident_detail_2.png)
          ![my screenshot](images/aisera-major_incident_detail_3.png)
          ![my screenshot](images/aisera-major_incident_detail_4.png)

Once Aisera detects a major incident:

- It notifies the designated persona on the channel of their choice.
- Aisera creates an incident in Datadog for Datadog Admins to have a visibility of the

Aisera can require approvals before a remediation task can run.

Aisera can also run any remediation task (Any API call/ Shell/Python/Powershell script/ Terraform manifests etc..), leveraging the no-code platform it provides.

## Data Collected

> One you run the datasource, Aisera will collect the metrics,alerts and logs  from datadog.

Reviewer: This tab will only render in-app if you are sending metrics into Datadog. I'd recommend adding this information under the Overview section with a `###` header

> One you run the datasource, Aisera will collect the metrics,alerts and logs  from datadog.

Reviewer: ```suggestion
Once you run the datasource, Aisera will collect metrics, alerts and logs from Datadog.
```

# Agent Check: Aisera

## Overview

>  [Aisera's][1] domain-agnostic [AIOps][2] platform identifies complex behaviors and patterns while analyzing correlations and causality on alerts and incidents across applications, services, and systems. 

Reviewer 3: ```suggestion
[Aisera's][1] domain-agnostic [AIOps][2] platform identifies complex behaviors and patterns while analyzing correlations and causality on alerts and incidents across applications, services, and systems. 
```

      11. **Change Request**: Any change request which caused this major incident.
      12. **Timeline**: A timeline of the incident/alerts correlation
      13. **Root Cause**: A root cause generated by the Aisera LLM model
      14. **Causal Graphs**: Graph of business services
      15. **Logs and Trace Analysis**
      16. **Major Incident Connectivity Graph**: A visual representation of how users are impacted by specific alerts/services/infrastructure
      17. **Score over Time**: AMI scores over time
      18. Outage Probability over Time
      19. **Major Incident Correlation Evidence**
          ![my screenshot](images/aisera-major_incident_detail_1.png)
          ![my screenshot](images/aisera-major_incident_detail_2.png)
          ![my screenshot](images/aisera-major_incident_detail_3.png)
          ![my screenshot](images/aisera-major_incident_detail_4.png)
8. Once Aisera detects a major incident:
- It notifies the designated persona on the channel of their choice.
- Aisera creates an incident in Datadog for Datadog Admins to have a visibility of the
10. We can also put Approvals in place so that before we approvals before any remediation step can run.

> 11. Aisera can also run any remediation task (Any API call/ Shell/Python/Powershell script/ Terraform manifests etc..) leveraging the No-code platform it provides.

Reviewer 3: Making a note that this needs to be updated to reflect OAuth instructions. See an example here:

https://github.com/DataDog/integrations-extras/blob/master/ably/README.md

> 8. Once Aisera detects a major incident:
> - It notifies the designated persona on the channel of their choice.
> - Aisera creates an incident in Datadog for Datadog Admins to have a visibility of the
> 10. We can also put Approvals in place so that before we approvals before any remediation step can run.
> 11. Aisera can also run any remediation task (Any API call/ Shell/Python/Powershell script/ Terraform manifests etc..) leveraging the No-code platform it provides.

Reviewer 2: ```suggestion

Once Aisera detects a major incident:

- It notifies the designated persona on the channel of their choice.
- Aisera creates an incident in Datadog for Datadog Admins to have a visibility of the

Aisera can require approvals before a remediation task can run.

Aisera can also run any remediation task (Any API call/ Shell/Python/Powershell script/ Terraform manifests etc..), leveraging the no-code platform it provides.
```
Note that this sentence is still missing info at the end: "Aisera creates an incident in Datadog for Datadog Admins to have a visibility of the".

## Data Collected

One you run the datasource, Aisera will collect the metrics,alerts and logs  from datadog.

### Service Checks

Aisera does not include any service checks.

### Events

Aisera does not include any events.

## Troubleshooting

Need help? Contact [support@aisera.com](mailto:support@aisera.com).

[1]: https://aisera.com
[2]: https://aisera.com/products/aiops/
[3]: https://docs.datadoghq.com/account_management/api-app-keys/#application-keys
[4]: https://poc0.login.aisera.cloud/#YWlzZXJhLmV4dGVybmFsU3lzdGVtcy5leHRlcm5hbFN5c3RlbXNWaWV3
[5]: https://aisera.com/contact/
[6]: https://github.com/DataDog/integrations-extras/blob/master/aisera/metadata.csv
[7]: https://docs.datadoghq.com/account_management/api-app-keys/#add-an-api-key-or-client-token
[8]: https://docs.datadoghq.com/account_management/api-app-keys/#add-application-keys

> 
> 
> [6]: https://github.com/DataDog/integrations-extras/blob/master/aisera/metadata.csv
> [7]: https://docs.datadoghq.com/account_management/api-app-keys/#add-an-api-key-or-client-token
> [8]: https://docs.datadoghq.com/account_management/api-app-keys/#add-application-keys

Reviewer 2: ```suggestion
```
Removing the duplicate links.
